compose/ui/ui/src/uikitMain/kotlin/androidx/compose/ui/viewinterop/UIKitInteropTransaction.uikit.kt

```diff
@@ -38,51 +27,42 @@ internal typealias UIKitInteropAction = () -> Unit
  * A transaction containing changes to UIKit objects to be synchronized within [CATransaction] inside a
  * renderer to make sure that changes in UIKit and Compose are visually simultaneous.
  * [actions] contains a list of lambdas that will be executed in the same CATransaction.
- * [state] defines if rendering strategy should be changed along with this transaction.
+ * [isInteropActive] defines if rendering strategy should be changed along with this transaction.
  */
 internal interface UIKitInteropTransaction {
     val actions: List<UIKitInteropAction>
-    val state: UIKitInteropState
-}
-
-internal fun UIKitInteropTransaction.isEmpty() =
-    actions.isEmpty() && state == UIKitInteropState.UNCHANGED
+    val isInteropActive: Boolean
```

Comment:
Why changes in interop required here?

Reply:
Because it's simpler, than to count interop sessions starts.